OPCB/insurgency/common/server/AI/functions.sqf

```diff
@@ -278,10 +278,10 @@ createRoofGun = {
 	_ai moveInGunner _gun;	
 	_grp setFormDir _dir;
 	_grp deleteGroupWhenEmpty true;
+
+	[_gun] call CHAB_fnc_serverGroups;
 	
-	_gun enableWeaponDisassembly false;
-	_gun setVariable ["ace_dragging_canDrag", false, true];
```

Comment:
Since you say this doesn't work, can you add the same fixes you made for other statics?